src/sql/azdata.d.ts

```diff
@@ -26,13 +26,21 @@ declare module 'azdata' {
 			groupId: string;
 			saveProfile: boolean;
 			azureTenantId?: string;
+			options: { [name: string]: any };
 
 			static createFrom(options: any[]): ConnectionProfile;
 		}
 
 		/**
 		 * Get the current connection based on the active editor or Object Explorer selection
-		*/
+		 */
 		export function getCurrentConnection(): Thenable<ConnectionProfile>;
+
+		/**
+		 * Get known connection profiles
+		 * @param activeConnectionsOnly Indicates whether only get the active connections, default value is false.
```

Comment:
>Only Indicates whether only get the active connections, default value is false. [](start = 29, length = 79)

Space after param name and fix grammer - i.e. 

Indicates whether to only get the active connections, default value is false.

Reply:
There is already a space after the parameter, do you mean more spaces?